payment-request/payment-request-constructor.https.html

```diff
@@ -368,21 +376,46 @@
   assert_equals(
     request.shippingOption,
     null,
-    "request.shippingOption must be null"
+    "shippingOption must be null, as requestShipping is missing"
```

Comment:
This is inconsistent with test behavior. Might append the original test below the assertion check.
`
const request2 = new PaymentRequest(defaultMethods, details, {requestShipping: true});
assert_equals( request.shippingOption, null, `request.shippingOption must be null`);
`